wire-ios/Wire-iOS.xcodeproj/project.pbxproj

```diff
@@ -10667,6 +10669,14 @@
 				minimumVersion = 1.0.0;
 			};
 		};
+		407831AB2AC4636F005C2978 /* XCRemoteSwiftPackageReference "DifferenceKit" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/wireapp/DifferenceKit/";
+			requirement = {
+				kind = upToNextMinorVersion;
```

Comment:
Could be forced to use commit
f9e042ddefdbf045d394ff211f1c247f1d2e488a
instead of version